chardevice_test.go

```diff
@@ -7,6 +7,8 @@ var (
 	deviceCharDeviceBackendSocket   = "-chardev socket,id=serial0,path=/tmp/console.sock,server=on,wait=off"
 	deviceCharDeviceBackendStdioMux = "-chardev stdio,id=serial0,mux=on,signal=off"
 	deviceCharDeviceMultiple        = "-chardev socket,id=serial0,path=/tmp/console.sock,server=on,wait=off -chardev socket,id=monitor0,path=/tmp/monitor.sock,server=on,wait=off"
+	deviceCharDevicePCIDriver		= "-serial none -chardev socket,id=serial0,path=/tmp/console.sock,server=on,wait=off -device pci-serial,id=pciser0,chardev=serial0"
+	deviceCharDevicePCIDriver2x		= "-serial none -chardev socket,id=serial0,path=/tmp/console.sock,server=on,wait=off -device pci-serial-2x,id=pciser0,chardev1=serial0"
 )
```

Comment:
whitespace formatting looks  off, shouldn't the  '=' be aligned?

Reply:
https://github.com/project-machine/qcli/pull/3

Reply:
Ah sorry will rebase